docs/getting-started/installation.md

```diff
@@ -0,0 +1,179 @@
+# Installation
+
+## Running JanusGraph inside a Docker container
+
+For virtualization and easy access, JanusGraph provides a [Docker image](https://hub.docker.com/r/janusgraph/janusgraph).
+Docker makes it easier to run servers and clients on a single machine without dealing with multiple installations.
+For instructions on installing and using Docker, please refer to the [docker guide](https://docker.com/get-started).
+Let's try running a simple JanusGraph instance in Docker:
+```bash
+$ docker run -it -p 8182:8182 janusgraph/janusgraph:latest
+```
+We run the image interactively and request Docker to make the container's port `8182` available for us to see.
+The server may need a few seconds to start up so be patient and wait for the corresponding log messages to appear.
+
+??? note "Full example log"
```

Comment:
I don't think there is much value in a full log. Please remove everything that isn't really interesting as it just adds noise. You can simply replace it with `...`. See also how this looks like [in the TinkerPop reference docs](http://tinkerpop.apache.org/docs/current/reference/#gremlin-server-docker-image).

Reply:
Wasn't this what you requested [earlier](https://github.com/JanusGraph/janusgraph/pull/1909#discussion_r360343009)?

Reply:
You mentioned there that the user should _wait for the corresponding log messages to appear_ without saying which log messages. I found that a bit confusing so I suggested to show which log messages you're referring to. Showing all log messages is however also not really helpful in my opinion as it's just too much information that isn't really helping.

I think that it helps to show a bit log output that can be expected when the server starts successfully like `Channel started at port 8182` as the last log message.

Reply:
Alright, now I see your point. I fixed this by removing most of the less interesting output lines.